demo/src/screens/componentScreens/FaderScreen.tsx

```diff
@@ -10,6 +11,7 @@ const itemWidth = 100;
 const itemHeight = 100;
 const tintColor = undefined;
 
+// @ts-ignore
```

Comment:
did you read the TS error here? 
it's actually right (: the condition will have return false...

Reply:
I know, it depends on the "screen configuration" above (numberOfItems ... tintColor), we wanted to keep the screen simple (no buttons etc) so I've put the configuration in code only.
But I did want the screen to adjust properly to the configuration change, so we get an error because of this "constant configuration".